frontend/src/scenes/onboarding/OnboardingProductIntroduction.tsx

```diff
@@ -0,0 +1,250 @@
+                    volume discounts
+                </Link>{' '}
+                automatically applied.
+            </>
+        ),
+        <>
+            Set <b>usage limits as low as $0</b> so you never get an unexpected bill.
+        </>,
+        <>Pay only for what you use.</>,
+        <>
+            Or, stay on our generous free plan if you'd like - you still get{' '}
+            <b>
+                {convertLargeNumberToWords(
+                    currentAndUpgradePlans.currentPlan.free_allocation ||
+                        currentAndUpgradePlans.downgradePlan.free_allocation ||
+                        0,
+                    null
+                )}{' '}
+                {product.unit}s free
+            </b>{' '}
+            every month.
+        </>,
+    ]
+
+    return (
+        <div className="w-full max-w-screen-xl">
+            <h3 className="mb-4 text-2xl font-bold">Usage-based pricing that only scales when you do</h3>
+            <ul className="pl-2 flex flex-col gap-y-1">
+                {pricingListItems.map((item, i) => (
+                    <li className="flex gap-x-2 items-start" key={`pricing-item-${i}`}>
+                        <IconCheck className="inline-block text-success shrink-0 mt-1" />
+                        <span>{item}</span>
+                    </li>
+                ))}
+            </ul>
+            <ProductPricingModal
+                product={product}
+                modalOpen={isPricingModalOpen}
+                planKey={planForStats.plan_key}
+                onClose={toggleIsPricingModalOpen}
+            />
+        </div>
+    )
+}
+
+export function OnboardingProductIntroduction(): JSX.Element | null {
+    const { product } = useValues(onboardingLogic)
+    const websiteSlug: Partial<Record<ProductKey, string>> = {
+        [ProductKey.SESSION_REPLAY]: 'session-replay',
+        [ProductKey.FEATURE_FLAGS]: 'feature-flags',
+        [ProductKey.SURVEYS]: 'surveys',
+        [ProductKey.EXPERIMENTS]: 'experimentation',
+        [ProductKey.PRODUCT_ANALYTICS]: 'product-analytics',
+    }
+
+    return product ? (
+        <>
+            <div className="unsubscribed-product-landing-page -m-4">
+                <header className="bg-primary-alt-highlight border-b border-t border-border flex justify-center p-8">
+                    <div className="grid md:grid-cols-2 items-center gap-8 w-full max-w-screen-xl">
+                        <div className="">
+                            <h2 className="text-2xl font-bold">{product.name}</h2>
+                            <h3 className="text-4xl font-bold">{product.headline}</h3>
+                            <p>{product.description}</p>
+                            <GetStartedButton product={product} />
+                        </div>
+                        <aside className="text-right my-2 hidden md:block">
+                            <img src={product.image_url || undefined} className="max-w-96" />
```

Comment:
Maybe better to not show the image if there isn't an `image_url` defined. 
<img width="1240" alt="Screenshot 2024-01-25 at 1 08 06 PM" src="https://github.com/PostHog/posthog/assets/21014901/213400e4-8dc4-4592-ab31-a6340a498847">